### DISCUSSION TITLE
Dudas

### DISCUSSION THREAD

**Discussion by Dany-Molina** [Launch-X-Latam/MisionFrontEnd#446] [Team Posts]
Hola que tal tengo unas dudas este yo estoy en el curso de ciberseguridad en el trabajo 
![image](https://user-images.githubusercontent.com/114841802/195091811-b09fb803-02c4-41c6-b91e-ed9aed1d19c1.png)
pero ya en GITHUB PUES no se si estoy bien no hecho nada porque no se que se debe de hacer
![image](https://user-images.githubusercontent.com/114841802/195092170-c703c0dc-a8cc-4902-9fe3-5d03b2082450.png)
y solo me aparece en repositorios MISION FRONTREND

**Comment by Dany-Molina** [comment]
![image](https://user-images.githubusercontent.com/114841802/195093712-09dedad4-330a-482c-8b42-0c462c3a69ef.png)
QUE SERIA ESE

**Comment by EsroblasaLX** [comment]
Hola, Tienes que validar que estes correctamente en el bloque de el Launch

**Comment by Amhedriel** [comment]
Te recomiendo que hagas un fork del repositorio y comiences a repasar.

**Comment by MauryPani** [comment]
GitHub es unicamente para Lunch X, si ya te aparece que estas dentro de la organización y tienes acceso al repositorio de FronEnd-Mision hiciste bien los pasos. Ahora solo entra al repo y lee lo que contiene.

Para el curso de ciberseguridad en el trabajo es otro y no tienen nada que ver, ese es unicamente por Microsoft teams.
